pkg/handlers/v1/nexposeassetproducer_test.go

```diff
@@ -70,12 +73,12 @@ func TestNexposeAssetProducerHandlerMultipleAssets(t *testing.T) {
 	close(assetChan)
 	close(errChan)
 
-	assetFetcher.EXPECT().FetchAssets(gomock.Any(), siteID, scanID).Return(assetChan, errChan)
+	mockAssetFetcher.EXPECT().FetchAssets(gomock.Any(), siteID, scanID).Return(assetChan, errChan)
```

Comment:
Task: the tests will also have to be tweaked to work with the `AssetFetcher` interface change. The good news is that they'll be simpler now.

Reply:
done